adafruit_bme680.py

```diff
@@ -104,7 +111,7 @@
 )
 
 
-def _read24(arr):
+def _read24(arr: bytearray) -> float:
```

Comment:
The type for `arr` can actually be further generalized to `circuitpython_typing.ReadableBuffer` which may help if someone ever wants to use `bytes` or something:

```suggestion
def _read24(arr: ReadableBuffer) -> float
```

Just make sure to use `from circuitpython_typing import ReadableBuffer` after the `import typing` line at the top!